installation-bundle/src/Database/Installer.php

```diff
@@ -165,9 +179,19 @@ private function compileCommands(): void
             foreach ($GLOBALS['TL_HOOKS']['sqlCompileCommands'] as $callback) {
                 $return = \System::importStatic($callback[0])->{$callback[1]}($return);
             }
+
+            // make sure commands added via the hook are also appended to the command order
+            foreach ($return as $commandSet) {
+                foreach ($commandSet as $hash => $sql) {
+                    if (!isset($order[$hash])) {
```

Comment:
Shouldn’t this be

```suggestion
                    if (!\in_array($hash, $order, true)) {
```

instead?

Reply:
true! good catch